tests/integration/targets/aci_cloud_region/tasks/main.yml

```diff
@@ -0,0 +1,66 @@
+# Test code for the ACI modules
+# Copyright: (c) 2020, Cindy Zhao (@cizhao) <cizhao@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Test that we have an ACI APIC host, ACI username and ACI password
+  fail:
+    msg: 'Please define the following variables: aci_hostname, aci_username and aci_password.'
+  when: aci_hostname is not defined or aci_username is not defined or aci_password is not defined
+
+# CLEAN ENVIRONMENT
+- name: Set vars
+  set_fact: 
+    mso_info: &mso_info
+      host: '{{ aws_aci_hostname }}'
+      username: '{{ aws_aci_username }}'
+      password: '{{ aws_aci_password }}'
+      validate_certs: '{{ aci_validate_certs | default(false) }}'
+      use_ssl: '{{ aci_use_ssl | default(true) }}'
+      use_proxy: '{{ aci_use_proxy | default(true) }}'
+      output_level: '{{ aci_output_level | default("info") }}'
+
+- name: Query all regions
+  aci_cloud_region:
+    <<: *mso_info
+    cloud_provider_profile_vendor: 'aws'
+    state: query
+  register: query_all
+
+- name: Verify query_all
+  assert:
+    that:
+    - query_all is not changed
+    - query_all.current.0.cloudProvP.attributes.dn == "uni/clouddomp/provp-aws"
+    - query_all.current.0.cloudProvP.attributes.vendor == "aws"
+    - query_all.current.0.cloudProvP.children | length == 17
```

Comment:
Change the check to be more than lets say 10 just in case the number of region change in the future.